src/plot_erp.jl

```diff
@@ -96,27 +63,14 @@ function plot_erp!(
     categorical_color = true,
     categorical_group = true,
     stderror = false, # XXX if it exists, should be plotted
-    pvalue = nothing,
-    butterfly = false,
-    topolegend = nothing,
-    topomarkersize = nothing,
-    topowidth = nothing,
-    topoheight = nothing,
-    topopositions_to_color = nothing,
-    topohalign = 0.05,
-    topovalign = 0.95,
-    topoaspect = 1,
+    significance = nothing,
     mapping = (;),
     kwargs...,
 )
     config = PlotConfig(:erp)
     config_kwargs!(config; mapping, kwargs...)
-    if butterfly
-        config = PlotConfig(:butterfly)
-        config_kwargs!(config; mapping, kwargs...)
-    end
-
-    plot_data = deepcopy(plot_data) # XXX why?
+    config_kwargs!(config; mapping, layout = (; show_legend = false))
```

Comment:
I might miss context here but two questions
1) could this be combined with L71?
2) is it still possible to do `show_legend = true`?

Reply:
The problem with show_legend that it is doesn't give a control over legend and colorbar (if I remember correctly), which cause that overlaying bug
That's why I wanted to get rid of it

Reply:
did you get rid of show_legend everywhere? or is it only here? is this documented?

Reply:
only here
we can deleted it or we can create workaround as in the following commit where show_legend is used for calling use_legend and use_colorbar